test/built-ins/RegExp/prototype/Symbol.search/success-return-val.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 21.2.5.9
+description: Return value following successful match
+info: >
+    [...]
+    14. Return Get(result, "index").
```

Comment:
Having another test to verify getting `index` from a **result** object returned from a custom `exec` would make it more comprehensive.